### PR TITLE
Create BSLet177.xml

### PR DESCRIPTION
**Pull request by karljonaskarlsson**

### CODE REVIEW

**Review by thea-m — APPROVED** [review]

File: LondonBibleSociety/BSLet177.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                     </altIdentifier>
+                     <altIdentifier>
+                        <idno>Cowley IV</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+               <msContents>
+                  <summary/>
+                  <msItem xml:id="ms_i1">
+                     <title ref="LIT1961Mashaf"/>
+                     <textLang mainLang="gez"/>
```
Comment (SophiaD-M) [inline LondonBibleSociety/BSLet177.xml:42]: above in title statement you wrote "Sa'atat with Psalms of David", should there then here be two points for msContent?

Comment (karljonaskarlsson) [reply LondonBibleSociety/BSLet177.xml:42]: I was thinking that the Psalms of David are an integral part of this Sa'atat

Comment (SophiaD-M) [reply LondonBibleSociety/BSLet177.xml:42]: I think this is an open research question

Comment (karljonaskarlsson) [reply LondonBibleSociety/BSLet177.xml:42]: I agree, and I wouldn't add the Psalter as a msContent

Comment (karljonaskarlsson) [reply LondonBibleSociety/BSLet177.xml:42]: To clarify: The Psalter is not to be found from the beginning to the end, but there are different Sa'atat prayers in between. So it is quite well integrated into the manuscript.

**Pull request merged**